NuRadioReco/modules/trigger/radiant_aux_trigger.py

```diff
@@ -0,0 +1,204 @@
+        """
+        calculats a power integration over a given time window
+
+        Parameters
+        ----------
+        trace: array of floats
+            the trace
+        window: float
+            the integration windown in ns
+
+        Returns
+        -------
+        integrated power: array of floats
+            the integrated power trace
+        """
+        dt = 1. / self.sampling_rate
+        i_window = int(int_window / dt)
+        power = trace ** 2
+        int_power = np.convolve(power, np.ones(i_window, dtype=int), 'valid') * dt
+        return int_power
+
+    def get_trigger_response(self, trace, threshold_sigma, int_window):
+        """
+        checks if which sample bins of a trace satisfies the trigger condition
+
+        Parameters
+        ----------
+        trace: array of floats
+            the signal trace
+        threshold: float
+            the threshold as a factor of rms noise
+        window: float
+            the integration window in ns
+
+        Returns
+        -------
+        triggered bins: array of bools
+            the bins where the trigger condition is satisfied
+        threshold: float
+            the threshold value in power integrated units (V^2/ns)
+        """
+        filtered_trace = self.get_filtered_trace_trigger(trace)
+
+        power_int_trace = self.get_power_int_trace(filtered_trace, int_window)
+        threshold = self.get_threshold(filtered_trace, threshold_sigma, int_window)
+        triggered_bins = np.array(power_int_trace) > threshold
+        return (triggered_bins, threshold)
+
+    @register_run()
+    def run(self, evt, station, det, threshold_sigma=30, int_window=11, coinc_window=60, number_coincidences=2, triggered_channels=None, trigger_name='radiant_aux_trigger'):
+        """
+        Simulates the radiant auxiliar trigger based on a schottky diode. The parametrization is based on lab measurments of the full radiant board and fitted with a tanh function.
+        The trigger is applied to the channels specified in triggered_channels. The coincidence window indicates the time within different channels should have triggered, 
+        in order to trigger the station. The number of coincidences specifies the number of channels that need to have triggered within the coincidence window.
+
+        Parameters
+        ----------
+        evt: Event
+            Event to run the module on
+        station: Station
+            Station to run the module on
+        det: Detector
+            The detector description
+        number_coincidences: int
+            number of channels that are required in coincidence to trigger a station
+        triggered_channels: array of ints or None
+            channels ids that are triggered on, if None trigger will run on all channels
+        coinc_window: float
+            time window in which number_coincidences channels need to trigger
+        trigger_name: string
+            a unique name of this particular trigger
```

Comment:
I think you are missing `threshold_sigma` and `int_window`. Also, `coinc_window` is out of order. Double check this and make sure that everything is included, and it is in the same order as the parameters in the function signature.

Reply:
Thanks for checking so carefully. I was really in a rush when I wrote this....